plugin/storage/es/samplingstore/storage.go

```diff
@@ -14,14 +14,14 @@ import (
 
 	"github.com/jaegertracing/jaeger/cmd/collector/app/sampling/model"
 	"github.com/jaegertracing/jaeger/pkg/es"
+	"github.com/jaegertracing/jaeger/pkg/es/config"
 	"github.com/jaegertracing/jaeger/plugin/storage/es/samplingstore/dbmodel"
 )
 
 const (
-	samplingIndex        = "jaeger-sampling"
-	throughputType       = "throughput-sampling"
-	probabilitiesType    = "probabilities-sampling"
-	indexPrefixSeparator = "-"
+	samplingIndex     = "jaeger-sampling"
```

Comment:
to follow the convention from span store
```suggestion
	samplingIndexBaseName     = "jaeger-sampling"
```